content/applications/inventory_and_mrp/manufacturing/management/scrap_manufacturing.rst

```diff
@@ -0,0 +1,106 @@
+inventory. For more information about virtual locations, see the documentation about the different
+types of :ref:`locations <manufacturing/management/difference-warehouse-location>`.
+
+.. tip::
+   Scrap orders can be viewed by navigating to :menuselection:`Inventory --> Operations --> Scrap`.
+   Each scrap order shows the date and time the order was created, along with the product and
+   quantity that was scrapped.
+
+   To view the total quantity of each item scrapped, navigate to :menuselection:`Inventory -->
+   Configuration --> Locations`. Remove the :guilabel:`Internal` filter from the
+   :guilabel:`Search...` bar to display virtual locations. Finally, select the
+   *Virtual Locations/Scrap* location.
+
+Odoo *Manufacturing* allows for both components and finished products to be scrapped within a
+manufacturing order. The specific type of item that can be scrapped during a manufacturing order
+depends on the stage of the manufacturing process.
+
+Scrap manufacturing components
+==============================
+
+To scrap components during the manufacturing process, begin by navigating to
+:menuselection:`Manufacturing --> Manufacturing Orders`, then select a manufacturing order or click
+:guilabel:`Create` to configure a new one. If a new manufacturing order is created, select a product
+from the :guilabel:`Product` drop-down menu, then click :guilabel:`Confirm`.
+
+Once the manufacturing order has been confirmed, a :guilabel:`Scrap` button appears at the top of
+the page. Click the button and a :guilabel:`Scrap` pop-up window appears.
+
+.. image:: scrap_manufacturing/scrap-button.png
+   :align: center
+   :alt: The scrap button on a manufacturing order.
+
+From the :guilabel:`Product` drop-down menu on the :guilabel:`Scrap` pop-up window, select the
+component that is being scrapped, then enter the quantity in the :guilabel:`Quantity` field.
+Finally, click :guilabel:`Done` to scrap the component.
+
+.. image:: scrap_manufacturing/scrap-pop-up.png
+   :align: center
+   :alt: The Scrap pop-up window.
+
+.. tip::
+   Before clicking :guilabel:`Mark As Done` on a manufacturing order, only the components of the
+   finished product can be scrapped, not the finished product itself. This is because Odoo recognizes
+   that the finished product cannot be scrapped before it has been manufactured.
+
+After scrapping a component, continue the manufacturing process using the required quantity of the
+component that was scrapped. The on-hand stock count for the component that was scrapped updates to
+reflect both the scrapped quantity and the quantity consumed during manufacturing.
+
+.. example::
+   If the manufacturing of Product X requires 10 units of Component Y, and five units of Component Y
+   were scrapped during the manufacturing process, the total quantity of Component Y consumed will be
+   15: 10 units used to manufacture Product X + five units scrapped.
```

Comment:
the use of numbers and words ("10" units and "five" units...) was used. I'd suggest picking one and using it throughout, for uniformity sake.